test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,26 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs) {
+    this->m_servoPwmOut = new PwmOut(servoPin);
+    this->m_servoPwmOut->period_ms(20);  //setting the PWM period to 20ms
+    this->m_servoRangeInDegrees = servoRangeInDegrees;
+    this->m_minPulsewidthInMs = minPulsewidthInMs;
+    this->m_maxPulsewidthInMs = maxPulsewidthInMs;
```

Comment:
It's better practice to use initializer lists (https://en.cppreference.com/w/cpp/language/constructor) to initialize member variables